src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java

```diff
@@ -0,0 +1,44 @@
+package com.streamforge.realm.twitch.oauth.dtos;
+
+import java.util.List;
+
+public class TwitchUserTokenDto {
+    private String access_token;
```

Comment:
Is it named not in CamelCase in order to make marshalling/unmarshalling easily?

Reply:
Yeah, that one only is the example, todo job is to re-map to normal naming and pass it to the rest client as entity.

Reply:
Could we export this DTO somehow to rest or vice versa - create it just on rest, send the payload to rest as it came from twitch and map it there?

Reply:
I think it's applicable, will do it